### PR TITLE
add real favicon

### LINKED ISSUE #93: add realfavicon files and markup

### PR DESCRIPTION
**Pull request by AucT**
Closes #93 

## Summary by CodeRabbit

- **New Features**
	- Updated the HTML head section for improved compatibility and visual representation across platforms.
	- Introduced a new `browserconfig.xml` for Microsoft browser settings, enhancing the app's appearance in the Windows Start menu.
	- Added a web manifest file to support Progressive Web Apps, improving usability on mobile devices.
	- Enhanced documentation with a guide on generating favicons.

These changes modernize the Beaver theme and enhance the overall user experience.

### CODE REVIEW

**Review by pftg — CHANGES_REQUESTED** [review]
@AucT please add documentation or add automatization scripts how can we generate those files

**Comment by AucT** [comment]
added in README.md

File: README.md [hunk 1]
```diff
@@ -69,3 +69,9 @@ bin/from_devto -f
 4. Open the dropdown with the "Run worflow" label and run it in the master branch.
 
 After the work flow is over, the article should appear on our blog and the canonical url on dev.to should be updated, it will point to the article on our blog.
+
+
+## How to generate new favicon, etc
+
+1. Use square svg (find or create)
```
Comment (pftg) [inline README.md]: Please add relative path to the path which we use as source from the repository

Comment (AucT) [reply README.md]: ++

File: themes/beaver/static/img/icons/logo-dark-square.svg
Comment (pftg) [inline themes/beaver/static/img/icons/logo-dark-square.svg:1]: please remove `dark` because it's not dark or mono colored

**Review by pftg — APPROVED** [review]

**Pull request merged**